src/Controller/CandidateController.php

```diff
@@ -123,4 +123,25 @@ public function showMyFavorite(): Response
     {
         return $this->render('candidate/showFavorite.html.twig', []);
     }
+
+    public function sidebarCandidate(): Response
+    {
+        /** @var User */
+        $user = $this->getUser();
+        $applications = $user->getCandidate()->getApplications();
+
+        $numberOfResponses = 0;
+
+        foreach ($applications as $application) {
+            if ($application->getApplicationStatus() !== "in-progress") {
```

Comment:
doit y avoir une constante qui traine pour "in-progress" (je crois que c'est Cedric qui l'a fait)